packages/cli/build.mjs

```diff
@@ -0,0 +1,83 @@
+import execa from 'execa';
+import { join } from 'node:path';
+import { copyFileSync, readFileSync, writeFileSync } from 'node:fs';
+import { esbuild } from '../../utils/build.mjs';
+
+function createConstants() {
+  console.log('Creating `constants.ts`');
+  const filename = new URL('src/util/constants.ts', import.meta.url);
+  const contents = `// This file is auto-generated
+export const GA_TRACKING_ID: string | undefined = ${envToString(
+    'GA_TRACKING_ID'
+  )};
+export const SENTRY_DSN: string | undefined = ${envToString('SENTRY_DSN')};
+`;
+  writeFileSync(filename, contents, 'utf8');
+}
+
+function envToString(key) {
+  const value = process.env[key];
+  if (!value) {
+    console.log(`- Constant ${key} is not assigned`);
+  }
+  return JSON.stringify(value);
+}
+
+// Read the secrets from GitHub Actions and generate a file.
+// During local development, these secrets will be empty.
+createConstants();
+
+// `vercel dev` uses chokidar to watch the filesystem, but opts-out of the
+// `fsevents` feature using `useFsEvents: false`, so delete the module here so
+// that it is not compiled by ncc, which makes the npm package size larger
+// than necessary.
+//await remove(join(dirRoot, '../../node_modules/fsevents'));
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (blocking):** ‏do we need this?